.github/ISSUE_TEMPLATE/bug-report---error-stack--.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report (`error-stack`)
+about: Create a report to help us improve `error-stack`
+title: "[BUG]"
+labels: A-error-stack, A-libs, C-bug
+assignees: TimDiekmann
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
It would be good, if a section for a code example could be added instead, a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)